packages/twenty-front/src/modules/object-record/cache/hooks/useCreateOneRecordInCache.ts

```diff
@@ -24,9 +24,13 @@ export const useCreateOneRecordInCache = <T extends ObjectRecord>({
   const apolloClient = useApolloClient();
 
   return (record: ObjectRecord) => {
+    const prefilledRecord = prefillRecord({
+      objectMetadataItem,
+      input: record,
+    });
     const recordGqlFields = generateDepthOneRecordGqlFields({
```

Comment:
I would split this in two methods: 
generateDepthOneRecordGqlFields(objectMetadataItem)
generateRecordGqlFields(objectMetadataItem, record)

Reply:
Will do in dedicated PR

Reply:
https://github.com/twentyhq/twenty/pull/10499